components/ProfileCard/ProfileCard.tsx

```diff
@@ -1,4 +1,8 @@
+// Copyright (c) Gridiron Survivor.
+// Licensed under the MIT License.
+
 'use client';
+import { JSX } from 'react';
```

Comment:
nit: use the type keyword


```suggestion
import type { JSX } from 'react';
```